docs/search/copilot.md

```diff
@@ -53,29 +73,57 @@ In the scenario depicted in the video, we leverage Copilot to investigate a secu
 1. Add the security `groupid` to the prompt to tabulate events containing that data.
 1. Focus on a specific access key, `ABCDEFGOYCM3PIKNOVRA`, noting `PutRolePolicy` events indicating permission elevation. Multiple AWS accounts are impacted.
 1. We conclude that the access key was used to modify permissions, providing the attacker with a potential network entry point. Remediation would involve disabling the key, blocking the source IP, and further log analysis.
-</details>
 
-## Step 1: Open Copilot
+### Security example
```

Comment:
I'm confused why you are including this video. It is the exact same example as the previous one, but presented without audio, and with visual highlighting of the steps.

Reply:
Same here! I've asked Bashyam about this in Slack.

Reply:
I've merged this branch (`copilot-vid`) into the `ai-machine-learning` branch. I'll work with Bashyam to sort out the duplicate video issue in [this PR](https://github.com/SumoLogic/sumologic-documentation/pull/3793).